install.sh

```diff
@@ -182,3 +182,14 @@ echo crew >> .git/info/sparse-checkout
 git fetch origin master
 git reset --hard origin/master
 echo "Chromebrew installed successfully and package lists updated."
+
+#check LD_LIBRARY_PATH on x86_64
+case "$architecture" in
+"x86_64")
+  (echo $LD_LIBRARY_PATH | grep '/usr/local/lib[^6]\|/usr/local/lib$' > /dev/null) || cat << EOF
```

Comment:
IMHO, it breaks several stuff.
 - how about ppl share .bashrc among several machines
 - how about ppl who don't want to add /usr/local/lib

you can read discussions in the web.  For example, https://softwareengineering.stackexchange.com/questions/241257/should-a-package-manager-modify-your-bashrc-file.